### PR TITLE
Add configure field for tcp filter report interval.

### PR DESCRIPTION
**Pull request by JimmyCYJ**
This is for solving issue# [539](https://github.com/istio/proxy/issues/539)

### CODE REVIEW

File: mixer/v1/config/client/client_config.proto [hunk 1]
```diff
@@ -124,4 +124,8 @@ message TcpClientConfig {
   // Quota specifications to generate quota requirements.
   // It applies on the new TCP connections.
   QuotaSpec connection_quota_spec = 5;
+
+  // Specify report interval in milliseconds to send report for TCP filter.
```
Comment (qiwzhang) [inline mixer/v1/config/client/client_config.proto]: to send periodical reports for long TCP connections.

Comment (JimmyCYJ) [reply mixer/v1/config/client/client_config.proto]: Comment is updated. Thanks!

File: mixer/v1/config/client/client_config.proto [hunk 2]
```diff
@@ -124,4 +124,9 @@ message TcpClientConfig {
   // Quota specifications to generate quota requirements.
   // It applies on the new TCP connections.
   QuotaSpec connection_quota_spec = 5;
+
+  // Specify report interval in milliseconds to send periodical reports for
+  // long TCP connections. If not specified, or 0 protobuf default value,
+  // the interval is 10 seconds.
+  int32 report_interval_ms = 6;
```
Comment (wora) [inline mixer/v1/config/client/client_config.proto]: Please use google.protobuf.Duration. Please always use Timestamp and Duration unless it is performance critical code.

Comment (JimmyCYJ) [reply mixer/v1/config/client/client_config.proto]: Thanks for the advice. I have a PR that updates this field in https://github.com/istio/api/pull/330

**Pull request merged**